Source/ChocolateyGui/Properties/Resources.de.resx

```diff
@@ -345,4 +345,211 @@
+    <value>Fehler beim Update zulassen</value>
+  </data>
+  <data name="PackageViewModel_FailedToUpdate" xml:space="preserve">
+    <value>Fehler beim Aktualisieren</value>
+  </data>
+  <data name="PackageViewModel_InstallingPackage" xml:space="preserve">
+    <value>Installiere Paket</value>
+  </data>
+  <data name="PackageViewModel_LoadingPackageInfo" xml:space="preserve">
+    <value>Lade Paketinformationen</value>
+  </data>
+  <data name="PackageViewModel_PinningPackage" xml:space="preserve">
+    <value>Fixiere Paketversion</value>
+  </data>
+  <data name="PackageViewModel_UninstallingPackage" xml:space="preserve">
+    <value>Deinstalliere Paket</value>
+  </data>
+  <data name="PackageViewModel_UpdatingPackage" xml:space="preserve">
+    <value>Aktualisiere Paket</value>
+  </data>
+  <data name="PackageView_ButtonPin" xml:space="preserve">
+    <value>Anheften</value>
+  </data>
+  <data name="RemoteSourceViewModel_FailedToLoad" xml:space="preserve">
+    <value>Fehler beim Herunterladen</value>
+  </data>
+  <data name="RemoteSourceViewModel_FeedSearchError" xml:space="preserve">
+    <value>Fehler beim Durchsuchen des Feeds</value>
+  </data>
+  <data name="RemoteSourceViewModel_LoadingPage" xml:space="preserve">
+    <value>Lade Paket {0} ...</value>
+  </data>
+  <data name="SettingsViewModel_DisplayName" xml:space="preserve">
+    <value>ChocolateyGUI - Einstellungen</value>
+  </data>
+  <data name="PackageViewModel_UnpinningPackage" xml:space="preserve">
+    <value>Erlaube Aktualisierung von Paket</value>
+  </data>
+  <data name="RemoteSourceViewModel_UnableToConnectToFeed" xml:space="preserve">
+    <value>Fehler beim Laden des Feeds: {0}.  Bitte überprüfe, dass der Feed erreichbar ist und versuche es nochmal.</value>
+  </data>
+  <data name="SettingsViewModel_RemovingSource" xml:space="preserve">
+    <value>Entferne Quelle...</value>
+  </data>
+  <data name="SettingsViewModel_SavingSource" xml:space="preserve">
+    <value>Speichere Quelle</value>
+  </data>
+  <data name="SettingsViewModel_SavingSourceLoading" xml:space="preserve">
+    <value>Speichere Quelle...</value>
+  </data>
+  <data name="SettingsViewModel_SourceMissingId" xml:space="preserve">
+    <value>Quelle benötigt eine Id!</value>
+  </data>
+  <data name="SettingsViewModel_SourceMissingValue" xml:space="preserve">
+    <value>Quelle benötigt einen Wert!</value>
+  </data>
+  <data name="SettingsView_Off" xml:space="preserve">
+    <value>Aus</value>
+  </data>
+  <data name="SettingsView_On" xml:space="preserve">
+    <value>Ein</value>
+  </data>
+  <data name="SettingsView_SourcesDisabled" xml:space="preserve">
+    <value>Deaktiviert</value>
+  </data>
+  <data name="VersionNumberProvider_VersionFormat" xml:space="preserve">
+    <value>Version: {0}</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_InstallFailedMessage" xml:space="preserve">
+    <value>Fehler beim Installieren von Paket "{0}", Version "{1}".
+Fehler: {2}{3}</value>

```

Comment:
Is that new line intentional? There a few more like this below (just look at the diff below)